samples/bot-sharepoint-list/csharp/SPFx_Bot/Bots/SpfxBot.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.BotBuilderSamples.SpfxBot.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.BotBuilderSamples.SpfxBot.Bots
+{
+    public class SpfxBot : ActivityHandler
```

Comment:
Please rename this to SharePointListBot. 
SPFx is a page and web part model that provides full support for client-side SharePoint development We are just using SharePoint list.